src/browser/documentation/help/keys.tsx

```diff
@@ -31,80 +31,201 @@ const category = 'browserUiCommands'
         </tr>
+        <tr>
+          <td />
+        </tr>
+      </tbody>
+    </table>
+
+    <table className="table-condensed table-help table-help--keys">
+      <thead>
+        <tr>
+          <th title="Field #1" className="padding30">
+            Editor action
+          </th>
+          <th title="Field #2" className="padding30">
+            Windows / Linux
+          </th>
+          <th title="Field #3" className="padding30">
+            Mac
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Execute current command</td>
+          <td>Ctrl + Enter</td>
+          <td>Cmd + Return</td>
+        </tr>
+
         <tr>
           <td>Previous command in history</td>
-          <td>
-            <div className="key code">
-              {isMac ? '<Cmd-Up-Arrow>' : '<Ctrl-Up-Arrow>'}
-            </div>
-          </td>
-          <td>
-            <div className="key code">{'<Up-Arrow>'}</div>
-          </td>
+          <td>Ctrl + Up Arrow</td>
+          <td>Cmd + Up Arrow</td>
         </tr>
+
         <tr>
           <td>Next command in history</td>
-          <td>
-            <div className="key code">
-              {isMac ? '<Cmd-Down-Arrow>' : '<Ctrl-Down-Arrow>'}
-            </div>
-          </td>
-          <td>
-            <div className="key code">{'<Down-Arrow>'}</div>
-          </td>
+          <td>Ctrl + Up Down</td>
+          <td>Cmd + Up Down</td>
         </tr>
+
         <tr>
-          <td>Switch to multi-line editing</td>
-          <td />
-          <td>
-            <div className="key code">{'<Shift-Return>'}</div>
-          </td>
+          <td>Search</td>
+          <td>Ctrl + F</td>
+          <td>Command + F</td>
         </tr>
         <tr>
-          <td />
+          <td>Increase Indent</td>
+          <td>Tab</td>
+          <td>Tab</td>
         </tr>
         <tr>
-          <th>Global actions</th>
-          <th />
-          <th />
+          <td>Decrease Indent</td>
+          <td>Shift + Tab</td>
+          <td>Shift + Tab</td>
         </tr>
         <tr>
-          <td>Change focus to editor</td>
-          <td>
-            <div className="key code">{printShortcut(FOCUS_SHORTCUT)}</div>
-          </td>
+          <td>Comment Out</td>
+          <td>Ctrl + /</td>
+          <td>Command + /</td>
         </tr>
         <tr>
-          <td>Toggle fullscreen editor</td>
-          <td>
-            <div className="key code">{printShortcut(FULLSCREEN_SHORTCUT)}</div>
-          </td>
+          <td>Comment In</td>
+          <td>Ctrl + /</td>
+          <td>Command + /</td>
         </tr>
         <tr>
-          <td />
+          <td>Undo</td>
+          <td>Ctrl + Z</td>
+          <td>Command + Z</td>
+        </tr>
+        <tr>
+          <td>Redo</td>
+          <td>Ctrl + Y</td>
+          <td>Shift + Command + Z or Command + Y</td>
+        </tr>
+        <tr>
+          <td>Increase font size</td>
+          <td>Ctrl + .</td>
```

Comment:
Not all of these commands are valid

Reply:
Removed font size commands from the list. Added information about F1 for a complete list.